src/experimental/riff-raff-yaml-file/index.test.ts

```diff
@@ -33,7 +176,7 @@ describe("The RiffRaffYamlFileExperimental class", () => {
 
     expect(() => {
       new RiffRaffYamlFileExperimental(app);
-    }).toThrowError("Unable to produce a working riff-raff.yaml file; missing 4 definitions");
+    }).toThrowError("Unable to produce a working riff-raff.yaml file; missing 1 definitions"); // Stack of media-service has no CODE stage
```

Comment:
Previously this was missing 4 definitions:

```
PASS src/experimental/riff-raff-yaml-file/index.test.ts
  ● Console

    console.log
      Unable to produce a working riff-raff.yaml file; missing 4 definitions (details below)

      at RiffRaffYamlFileExperimental.validateStacksInApp (src/experimental/riff-raff-yaml-file/index.ts:195:15)

    console.log
      For the class: MyApplicationStack

      at src/experimental/riff-raff-yaml-file/index.ts:197:17
          at Array.forEach (<anonymous>)

    console.log
      ┌───────────────┬────────────────────────────┐
      │    (index)    │         eu-west-1          │
      ├───────────────┼────────────────────────────┤
      │    deploy     │ { CODE: '✅', PROD: '✅' } │
      │ media-service │ { CODE: '❌', PROD: '✅' } │
      └───────────────┴────────────────────────────┘

      at src/experimental/riff-raff-yaml-file/index.ts:198:17
          at Array.forEach (<anonymous>)

    console.log
      For the class: MyDatabaseStack

      at src/experimental/riff-raff-yaml-file/index.ts:197:17
          at Array.forEach (<anonymous>)

    console.log
      ┌───────────────┬────────────────────────────┐
      │    (index)    │         eu-west-1          │
      ├───────────────┼────────────────────────────┤
      │    deploy     │ { CODE: '❌', PROD: '✅' } │
      │ media-service │ { CODE: '❌', PROD: '❌' } │
      └───────────────┴────────────────────────────┘

      at src/experimental/riff-raff-yaml-file/index.ts:198:17
```

The changes in this PR means only 1 definition is missing:

```
PASS src/experimental/riff-raff-yaml-file/index.test.ts
  ● Console

    console.log
      Unable to produce a working riff-raff.yaml file; missing 1 definitions (details below)

      at RiffRaffYamlFileExperimental.validateStacksInApp (src/experimental/riff-raff-yaml-file/index.ts:165:15)

    console.log
      ┌───────────────┬──────┬──────┐
      │    (index)    │ CODE │ PROD │
      ├───────────────┼──────┼──────┤
      │    deploy     │ '✅' │ '✅' │
      │ media-service │ '❌' │ '✅' │
      └───────────────┴──────┴──────┘

      at RiffRaffYamlFileExperimental.validateStacksInApp (src/experimental/riff-raff-yaml-file/index.ts:166:15)
```